src/pyTRLCConverter/markdown_converter.py

```diff
@@ -120,56 +240,235 @@ def convert_record_object(self, record: Record_Object, level: int) -> Ret:
         Returns:
             Ret: Status
         """
-        record_attributes = record.to_python_dict()
+        assert self._fd is not None
 
-        markdown_heading = self.markdown_create_heading(record.name, level + 1)
-        self._fd.write(markdown_heading)
+        self._write_empty_line_on_demand()
+        return self._convert_record_object(record, level, None)
 
-        column_titles = ["Attribute Name", "Attribute Value"]
-        markdown_table_head = self.markdown_create_table_head(column_titles)
-        self._fd.write(markdown_table_head)
+    def finish(self):
+        # lobster-trace: SwRequirements.sw_req_markdown_single_doc_mode
+        """Finish the conversion process.
+        """
 
-        for key, value in record_attributes.items():
-            if value is None:
-                value = "N/A"
+        # Single document mode?
+        if self._args.single_document is True:
+            assert self._fd is not None
+            self._fd.close()
+            self._fd = None
 
-            elif isinstance(value, list):
-                value = ", ".join(value)
+        return Ret.OK
 
-            markdown_table_row = self.markdown_append_table_row([key, value])
-            self._fd.write(markdown_table_row)
+    def _write_empty_line_on_demand(self) -> None:
+        # lobster-trace: SwRequirements.sw_req_markdown
+        """Write an empty line if necessary.
+        """
+        if self._empty_line_required is False:
+            self._empty_line_required = True
+        else:
+            self._fd.write("\n")
 
-        self._fd.write("\n")
+    def _get_markdown_heading_level(self, level: int) -> int:
+        # lobster-trace: SwRequirements.sw_req_markdown_record
+        """Get the Markdown heading level from the TRLC object level.
+            Its mandatory to use this method to calculate the Markdown heading level.
+            Otherwise in single document mode the top level heading will be wrong.
```

Comment:
Indent wrong for doc string, repeats through the file.

Reply:
The autodocstring extension always starts right after """.